packages/miniflare/src/index.ts

```diff
@@ -1169,7 +1169,13 @@ export class Miniflare {
 			);
 		}
 
-		return { services: servicesArray, sockets, extensions };
+		const autogates = [
+			// Enables Python support in workerd.
+			// TODO(later): remove this once this gate is removed from workerd.
+			"workerd-autogate-builtin-wasm-modules"
+		];
```

Comment:
Ideally we'd only enable this when Python support is requested in Wrangler. But I wasn't sure what the best way to do so would be, I also don't think it's a big deal to enable this for all (we will remove this autogate soon and the code behind it will be enabled for all anyway).